### PR TITLE
invitefriends rpc crud

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

File: go/service/invite_friends.go [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+		percentageChange    float64
+		showFire            bool
+		libkb.AppStatusEmbed
+	}
+	apiArg := libkb.APIArg{
+		Endpoint:    "invite_friends/num_invites_in_last_day",
+		SessionType: libkb.APISessionTypeNONE,
+	}
+	var res apiRes
+	err = mctx.G().API.GetDecode(mctx, apiArg, &res)
```
Comment (zapu) [inline go/service/invite_friends.go]: not handling the error here?

Comment (zapu) [reply go/service/invite_friends.go]: oh we don't want this to ever black bar? maybe it should be handled differently somehow then

Comment (heronhaye) [reply go/service/invite_friends.go]: no i just forgot lol, thanks

File: go/service/invite_friends.go [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	if err != nil {
+		return counts, err
+	}
+	return keybase1.InviteCounts{
+		InviteCount:      res.numInvitesInLastDay,
+		PercentageChange: res.percentageChange,
+		ShowFire:         res.showFire,
```
Comment (jzila) [inline go/service/invite_friends.go]: I also added a `res.showNumInvites` boolean flag so the server can control whether to show it.

**Pull request merged**